sparkle/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dust-tt/sparkle",
-  "version": "0.2.259",
+  "version": "0.2.260",
```

Comment:
bump again :-)

Reply:
And again 😅 , just kidding with merge my branch, and won't publish it.